test/manual/authentication/pytest_crypto.py

```diff
@@ -11,7 +11,9 @@
 def test_hashing():
     with open("/vagrant/hiera/settings.yaml", 'r') as stream:
         try:
-            cryptopath = yaml.load(stream)['crypto']['path']
+            yamlres = yaml.load(stream)
+            root = yamlres['general']['root']
+            cryptopath = root + yamlres['crypto']['path']
         except yaml.YAMLError as exc:
             print exc
```

Comment:
Need to 
`import logger`
but to do that properly with YAML, need something to log the potential error from importing the logger.